articles/spatial-anchors/how-tos/create-locate-anchors-unity.md

```diff
@@ -323,4 +324,4 @@ Learn more about the [Dispose](/dotnet/api/microsoft.azure.spatialanchors.clouds
     this.cloudSession.Dispose();
 ```
 
-[!INCLUDE [Next Steps](../../../includes/spatial-anchors-create-locate-anchors-next-steps.md)]
\ No newline at end of file
+[!INCLUDE [Next Steps](../../../includes/spatial-anchors-create-locate-anchors-next-steps.md)]
```

Comment:
This looks like a github merge issue? Do you know why it is removing and adding the same line? Would be best to remove this part of the PR so as to enable easier tracing/blame through old changes.